packages/graph/hash_graph/lib/graph/src/store/postgres/resolve/context.rs

```diff
@@ -87,4 +97,22 @@ impl<C: AsClient> PostgresContext for PostgresStore<C> {
             .await
             .attach_printable("could not read links")?)
     }
+
+    async fn read_active_links_by_source(
+        &self,
+        entity_id: EntityId,
+    ) -> Result<links::RecordStream, QueryError> {
+        Ok(links::read_active_links_by_source(&self.client, entity_id)
+            .await
+            .attach_printable("could not read links")?)
+    }
+
+    async fn read_active_links_by_target(
+        &self,
+        entity_id: EntityId,
+    ) -> Result<links::RecordStream, QueryError> {
+        Ok(links::read_active_links_by_target(&self.client, entity_id)
+            .await
+            .attach_printable("could not read links")?)
```

Comment:
Also, do you think it would be useful to attach the `EntityId` here, or do we do that anywhere up the error stack path?

Reply:
Added the `EntityId` in 41698fd1d6eccea3426f9555b7331c30324aa58a